.github/workflows/tests.yaml

```diff
@@ -31,7 +31,8 @@ jobs:
         node-version: '16'
         cache: 'yarn'
     - name: install dependencies
-      run: yarn install --frozen-lockfile
+      run: yarn cache clean && yarn install --frozen-lockfile
```

Comment:
we should not clean the cache: however it makes no sense to use it.

We may also want to add `--prefer-offline` flag to yarn install, to prefer using cached packages when possible.